### PR TITLE
Update charter.txt

### PR DESCRIPTION
**Pull request by mnkocher**

### CODE REVIEW

File: charter.txt [hunk 1]
```diff
@@ -17,7 +17,7 @@ Definition of done: when is a task complete?:
 Properly tested, checked into Github, meets all requirements, checked by teammate(s)
 
 Scrum meeting format/frequency and distribution of documentation tasks (meeting minutes, updating task board):
-~15 minutes 
+~15 minutes (Tue/Thr 9:45am)
```
Comment (amenahs) [inline charter.txt:20]: this time works for me

**Comment by jankow72** [comment]
This looks good.

**Review by amenahs — COMMENTED** [review]
this time works for me

**Review by jankow72 — APPROVED** [review]
This time is good

**Review by Catt-21 — APPROVED** [review]
This works for me too

**Pull request merged**